### ISSUE TITLE
raven fails

### ISSUE THREAD

**Issue by kazumaxneo** [RolandFaure/Hairsplitter#8]
Hi.

This is a great implementation.
When running Hairsplitter, however, it fails on the raven reassemblies after a while (attached picture).

$python hairsplitter.py -f SRR27955205.fastq -i metaflye_assembly.fasta -x ont -o outdir -t 20

I am not sure what the cause is, so I would appreciate it if you could include tiny dataset in the repository or write an example of some public data run if possible.

Kazuma Uesaka

![screen-shot](https://github.com/RolandFaure/Hairsplitter/assets/20102961/ecd8eef2-b6b1-4e5f-9caf-a14cd6c27326)

**Comment by RolandFaure** [comment]
Hello Kazuma,

I'm very glad to see you like GenomeTailor. If I understand well this issue is in the wrong repository and concerns GenomeTailor alone ? (the GenomeTailor version packed with HairSplitter should not have these problems)

The repository of GenomeTailor was a bit messy since I did not realize anyone actually used it except me, sorry for that. I cleaned it up, now it should work better. You were absolutely right about the test instance, I added it. 

Keep in mind that GenomeTailor is in beta version , that the reassembly module does not work very well for now and that I am actively developing it. I hope to have a significantly improved version by the end of next week. Nevertheless, all you comments are welcome to help improve GenomeTailor :smiley:, if you desire a particular feature it is the best time to say so!

**Comment by kazumaxneo** [comment]
Hi RolandFaure

Thank you !
I had a little concerned and cloned GenomeTailor repo. independently. 
That would caused this problem.
When I used GenomeTailor enclosed with HairSplitter as you commented, the problem did not happen
 (job finished successfully).

I noticed one another thing : "samtools consensus" commnad looks introduced from samtools v1.16.
So I think it would be better to use samtools > v1.16 in the install section of README  (currently 1.19 is the latest).

Take care.
